src/deploy/osp_deployer/profiles/csp.json

```diff
@@ -231,7 +239,7 @@
                   "valid_values": [
                   "true",
                   "false"
-               ],
+               ]
```

Comment:
The change on the above line introduces a syntax error to this file.  Need to undo this.